src/apiCalls.js

```diff
@@ -29,3 +29,5 @@ const fetchBookingsData = () => {
     .then(response => response.json())
     .catch(err => console.log('ERROR'))
 }
+
```

Comment:
Fun fact: You can condense your fetches into one function and pass in a variable when you call on that fetch. This can be a refactoring thing later. Something kinda like the below:

```
// fetch
const fetchData = (extension) => {
  return fetch(`http://localhost:3001/api/v1/${extension}`)
    .then((response) => {
      if (response.ok) {
        return response.json();
      } else {
        throw Error(response.statusText)
      }
    })
    .catch(err => console.log(err));
};

// in your Promise.all invoke the fetch like this:
let rooms = fetchData('rooms')
```